ui/src/timespan/calendar/CalendarPage.tsx

```diff
@@ -226,6 +226,13 @@ export const CalendarPage: React.FC = () => {
                             timeSpansResult.refetch(range);
                         }
                     }}
+                    views = {{
+                        timeGrid5Day: {
+                            type: 'timeGrid',
+                            duration: { days: 5 },
```

Comment:
Have you tried something similar to this?
https://stackoverflow.com/questions/33673487/fullcalendar-show-only-weekdays

Reply:
I did not find that `hiddenDays` key. This works.
Only thing is that since today is a weekend, it only shows 3 days for this week, where as before hiding Saturday Sunday it would show 5 starting from current day on the weekend.

I think I prefer having it show 5 days, but it definitely seems like a super easy setting to make configurable, since it's just a list of numbers.

Before:
![image](https://github.com/user-attachments/assets/727be806-3866-4b01-b3b5-6fa18ad82341)

After:
![image](https://github.com/user-attachments/assets/a3cd33c1-0a99-43af-b563-27028f6265fe)

Reply:
have you changed the duration to 7 days as in the example? for me it does work fine and currently shows 5 days.

Reply:
You're right once again, I will update the PR.